manifests/plugin/statsd.pp

```diff
@@ -8,7 +8,7 @@
   $deletegauges    = undef,
   $deletesets      = undef,
   $interval        = undef,
-  $timerpercentile = undef,
+  $timerpercentile = [],
```

Comment:
Setting a parameter to [] in class parameters confuses hiera. Please use pick() inside the manifest and default to undef here.